packages/blocks/src/api/test/registration.js

```diff
@@ -1528,6 +1528,74 @@ describe( 'blocks', () => {
 			);
 		} );
 
+		// Check the `usesContext` array is correct.
+		it( 'should reject invalid usesContext property', () => {
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				label: 'testing',
+				usesContext: 'should be an array',
+			} );
+			expect( console ).toHaveWarnedWith(
+				'Block bindings source usesContext must be an array.'
+			);
+		} );
+
+		it( 'should add usesContext when only defined in the client', () => {
```

Comment:
I believe this should read, "should add usesContext when only defined in the server".

Reply:
I made the change as part of [this commit](https://github.com/WordPress/gutenberg/pull/63513/commits/1d7160491a65c402a9b2ca5da6d4c1a87ee0b8c7).